playwright-e2e/dss/component/checkbox.ts

```diff
@@ -51,6 +51,10 @@ export default class Checkbox extends WidgetBase {
     }
   }
 
+  async isCheckboxDisabled(): Promise<boolean> {
```

Comment:
`isDisabled()` is clear and shorter name.

Reply:
I Agree. Thanks. Fixed!